server/tests/tram_track_graph_transformer/test_tram_track_graph_transformer.py

```diff
@@ -0,0 +1,190 @@
+from collections import deque
+from math import sqrt
+
+import overpy
+import pytest
+from networkx import DiGraph
+from pyproj import Geod
+from src.tram_track_graph_transformer import TramTrackGraphTransformer
+from src.tram_track_graph_transformer.node import Node
```

Comment:
I think you can combine these two imports:
```suggestion
from src.tram_track_graph_transformer import TramTrackGraphTransformer, Node
```